src/Curation/Curation.router.ts

```diff
@@ -91,45 +138,79 @@ export class CurationRouter extends Router {
       collectionId
     )
 
-    return curationService.getLatestById(collectionId)
+    const collection = await Collection.findOne(collectionId)
+    if (!isTPCollection(collection)) {
+      throw new HTTPError(
+        'Collection is not a third party collection',
+        { id: collectionId },
+        STATUS_CODES.badRequest
+      )
+    }
+
+    const publishedItems = await thirdPartyAPI.fetchItemsByCollection(
+      collection.third_party_id,
+      collectionId
+    )
```

Comment:
Certanly. Because of how we decided to approach this feature as a whole, it shouldn't be a problem in the short term and we should tackle it when paginating. I'll add a comment so we don't forget it